### ISSUE TITLE
A11y_SQLAzureDataStudio_SchemaCompare_SchemaCompareOptions_Keyboard: Keyboard focus is not shifted to the latest popped up window on the screen.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9409]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”
#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark; #DesktopApp;#VoiceOver;#MacOs;#MAS2.4.3;

Environment Details:
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

Additional Details:
Mac OS:10.15.3
MAS References:MAS2.4.3
Screen reader: VoiceOver

**Repro Steps:**
1.Launch Azure Data Studio application.
2.Connect to server with windows credentials.
3.Navigate to Extensions from left menu.
4.Search & install Schema Compare extension.
5.Navigate to Connections from left menu.
6.Expand the server connected in step 2.
7.Right click an on prem database and select Schema compare from Context menu.
8.Navigate till Select Source or Select Target & hit enter.
Provide all the details for Source & Target.
9.Navigate till Ok button & hit enter.
10.Navigate till Options menu from Schema compare Menu bar.
11.After hitting enter, check if focus order is proper in order.

**Actual:**
The focus is not shifted to the new window that pop ups on the screen after hitting enter on the options menu.

**Expected:**
As soon as any new window pop ups which contains any interactive element inside it the focus should shift inside that new screen.

**User Impact:**
Keyboard as well as screen reader users will get confused as the focus remains on the previous screen and not shifted to the latest screen that is visible to us.

**Attachment link reference:**
[9409_A11y_SQLAzureDataStudio_SchemaCompare_SchemaCompareOptions_Keyboard.zip](https://github.com/microsoft/azuredatastudio/files/4291306/9409_A11y_SQLAzureDataStudio_SchemaCompare_SchemaCompareOptions_Keyboard.zip)

**Comment by alanrenmsft** [comment]
issue: newly opened UI should have the keyboard focus

**Comment by manaswita-chougule** [comment]
Verified the issue in below environment :
Version: 1.41.1 (Universal)
Commit: 33dce95b152227c4f10f6251ab30690fe9895c0c
Date: 2023-01-28T00:49:19.454Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin arm64 22.3.0

The issue is fixed. Hence adding closed tag. 
#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-SQLADS-MAC-Feb2020;
